shlr/tree-sitter/lib/src/query.c

```diff
@@ -259,7 +259,7 @@ static void stream_scan_identifier(Stream *stream) {
  * CaptureListPool
  ******************/
 
-static CaptureListPool capture_list_pool_new() {
+static CaptureListPool capture_list_pool_new(void) {
```

Comment:
It's better if we don't touch these files but just work upstream. I will work with tree-sitter to include these fixes.